oneview/resource_server_profile.go

```diff
@@ -1185,6 +1185,19 @@ func resourceServerProfile() *schema.Resource {
 											},
 										},
 									},
+									"host_name": {
```

Comment:
can we change the block name to profile host here as it is for Golang and then attribute name as hostname?